packages/lxd/test/wait/ready.coffee

```diff
@@ -0,0 +1,120 @@
+
+nikita = require '@nikitajs/core/lib'
+{config, images, tags} = require '../../test'
+they = require('mocha-they')(config)
+
+return unless tags.lxd
+
+describe 'lxc.wait.ready', ->
+
+  they 'wait for the container to be ready', ({ssh})  ->
+    nikita
+      $ssh: ssh
+    , ({registry}) ->
+      await registry.register 'clean', ->
+        await @lxc.delete 
+          container: 'nikita-wait-1'
+          force: true
+      await registry.register 'test', ->
+        await @lxc.init
+          image: "images:#{images.alpine}"
+          container: 'nikita-wait-1'
+          start: true
+        {$status} = await @lxc.wait.ready 'nikita-wait-1'
+        $status.should.be.true()
+      try 
+        await @clean()
+        await @test()
+      finally
+        await @clean()
+  
+  return unless tags.lxd_vm
```

Comment:
The convention to place `unless tags.lxd_vm` is 
- only once around describe
- more and more, I place them after describe, with a new line